src/Avalonia.Controls/MenuBase.cs

```diff
@@ -31,13 +31,13 @@ public abstract class MenuBase : SelectingItemsControl, IFocusScope, IMenu
         /// Defines the <see cref="MenuOpened"/> event.
         /// </summary>
         public static readonly RoutedEvent<RoutedEventArgs> MenuOpenedEvent =
-            RoutedEvent.Register<MenuItem, RoutedEventArgs>(nameof(MenuOpened), RoutingStrategies.Bubble);
+            RoutedEvent.Register<MenuBase, RoutedEventArgs>(nameof(MenuOpened), RoutingStrategies.Bubble);
```

Comment:
I changed it since it looks like a copy-paste error, but I am not sure if this was intended?

Reply:
Yes I think this change is correct. Originally there was no `MenuBase` if I remember correctly, so that would explain it.